code/modules/admin/verbs/playsound.dm

```diff
@@ -44,9 +44,22 @@ GLOBAL_LIST_EMPTY(sounds_cache)
 	set name = "Play Local Sound"
 	if(!check_rights(R_SOUNDS))	return
 
-	log_admin("[key_name(src)] played a local sound [S]")
-	message_admins("[key_name_admin(src)] played a local sound [S]", 1)
-	playsound(get_turf(src.mob), S, 50, 0, 0)
+	var/sound/uploaded_sound = sound(S, repeat = 0, wait = 1, channel = CHANNEL_ADMIN)
+	uploaded_sound.priority = 250
+
+	GLOB.sounds_cache += S
+
+	if(alert("Are you sure?\nSong: [S]\nNow you can also play this sound using \"Play Local Sound\".", "Confirmation request" ,"Play", "Cancel") == "Cancel")
+		return
+
+	log_admin("[key_name(src)] played local sound [S]")
+	message_admins("[key_name_admin(src)] played local sound [S]", 1)
```

Comment:
`message_admins()` doesn't seem to have a second argument, unless I'm missing something? (Same on the other occurrences.)
https://github.com/ParadiseSS13/Paradise/blob/58d7054e92a2a67adf1e78da8ad524fdba8f2504/code/modules/admin/admin.dm#L5-L10
```suggestion
	message_admins("[key_name_admin(src)] played local sound [S]")
```